src/rcore.c

```diff
@@ -5456,7 +5456,7 @@ static void MouseButtonCallback(GLFWwindow *window, int button, int action, int
     gestureEvent.position[0].x /= (float)GetScreenWidth();
     gestureEvent.position[0].y /= (float)GetScreenHeight();
 
-    // Gesture data is sent to gestures system for processing
+    // Gesture data is sent to gestures-system for processing
```

Comment:
rcore.c line 5459: The original seems correct.  It is not used as an adjective as gestures-system would be.

Reply:
In that case it should not have s in gestures...

Reply:
That does sound better.  Is it accurate?

Reply:
gesture-system is ok